index.html

```diff
@@ -982,31 +982,30 @@ <h2>
         };
       </pre>
       <p>
-        <a>PushMessageData</a> objects have an associated [=byte sequence=] set on creation, which
-        is `null` if there was no data in the <a>push message</a>.
+        <a>PushMessageData</a> objects have an associated <dfn data-dfn-for=PushMessageData>bytes</dfn> (a
+        [=byte sequence=]), which is set on creation.
       </p>
       <p>
-        The <dfn>arrayBuffer()</dfn> method, when invoked, MUST return an {{ArrayBuffer}} whose
-        contents are |bytes|. Exceptions thrown during the creation of the {{ArrayBuffer}} object
-        are re-thrown.
+        The <dfn>arrayBuffer()</dfn> method steps are to return an {{ArrayBuffer}} whose contents
+        are <a>this</a>'s <a data-dfn-for=PushMessageData>bytes</a>. Exceptions thrown during the creation of
+        the {{ArrayBuffer}} object are re-thrown.
       </p>
       <p>
-        The <dfn>blob()</dfn> method, when invoked, MUST return a {{Blob}} whose contents are
-        |bytes| and |type| is not provided.
+        The <dfn>blob()</dfn> method steps are to return a new {{Blob}} object whose contents are
+        <a>this</a>'s <a data-dfn-for=PushMessageData>bytes</a>.
```

Comment:
```suggestion
        [=this=]'s [=PushMessageData/bytes=].
```

I think this works... same for others.

Reply:
It probably does, but that doesn't match the current editorial convention?

Reply:
I believe that convention is just because of this file is old and we don't have to maintain high verbosity 😛

Reply:
BTW is it intentional to remove the mention of type?

Reply:
Yeah, "_type_ is not provided" doesn't mean anything.